docs/tutorial/image_cls.md

```diff
@@ -149,9 +162,9 @@ output
  │   └── soc_system.dtb
  ├── models
  │   ├── lib (include trained model library)
- │   │   ├── libdlk_arm.so
- │   │   ├── libdlk_fpga.so
- │   │   └── libdlk_x86.so
+ │   │   ├── lib_arm.so
```

Comment:
@lm-lily 
These name like `libdlk_arm.so` are name in current master branch.
Not bug.

Reply:
Thank you @iizukak san, I cannot find libdlk_x86.so, instead I found lib_x86.so in the generated files. I shall ignore this?

Reply:
@lm-lily 
Please check
https://github.com/blue-oil/blueoil/pull/724